src/ServiceManagement/RemoteApp/Commands.RemoteApp.Tests/RemoteProgram/RemoteAppProgram.cs

```diff
@@ -0,0 +1,218 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Test.RemoteApp
+{
+    using Common;
+    using Microsoft.Azure.Management.RemoteApp.Cmdlets;
+    using Microsoft.Azure.Management.RemoteApp.Models;
+    using System;
+    using System.Collections.Generic;
+    using System.Management.Automation;
+    using VisualStudio.TestTools.UnitTesting;
+
+    // Publish-AzureRemoteAppProgram, Unpublish-AzureRemoteAppProgram
+
+    [TestClass]
+    public class RemoteAppProgramTest : RemoteAppClientTest
+    {
+
+        [TestMethod]
+        [Ignore]
```

Comment:
These tests are valid, however there are some issues with the way they are using the mock in the common test project, so we will fix them in the next iteration. Thus keeping it ignored for now.